docs/pages/application-access/okta/usersync.mdx

```diff
@@ -0,0 +1,97 @@
+---
+title: Okta User Synchronization
+description: Managing Okta users in Teleport
+---
+
+## User Synchronization
+
+Teleport uses two complimentary methods for synchronizing Teleport users with
+an upstream Okta Organization:
+ - the Okta integration [synchronization process](./architecture)
+ - provisioning via SCIM 
+
+On the whole, it doesn't matter to the wider Teleport system which method was
+used to import a user as both processes create identical Teleport user accounts. 
+
+#### Importing Okta Users
+
+During synchronization, the Okta integration will create Teleport user accounts 
+for all eligible Okta users, and clean up the Teleport accounts of any Okta users
+that have been deactivated. The list of "eligible Okta users" depends on the
+integration configuration. 
+
+If the Okta integration is configured to use a specific Okta application, then
+the Okta integration will consider any Okta user assigned to that Okta 
+application - either directly, or via group membership - as an "eligible user". 
+
+If no Okta application is to use is supplied, all enabled Okta organization
+users are considered "eligible".
+
+<Admonition type="note">
+All hosted Okta integrations are configured to restrict their pool of users to
+a specific Okta application.
+
+The integration [enrolment process](./hosted-guide/) creates an Okta SAML
+Application to use as the identity provider for a new
+[SAML Authentication Connector](../../access-controls/sso/okta/). The enrolment
+process automatically configures the Okta integration to use this same
+application, giving the Okta administrator a single point of integration to
+administer and secure.
+</Admonition>
+
+### Okta Profile to Teleport User
+
+The Teleport users created by the synchronization process or SCIM provisioning
+all inherit their username from the upstream Okta organization, and are given a
+default role of `requester`. 
```

Comment:
I think this is no longer true, right @mdwn? We give them `okta-requester` role?

Reply:
Once https://github.com/gravitational/teleport.e/pull/3454 is through, yes. Though reading this doc I realize I probably missed a few places.